cli/lib/tasks/download.ts

```diff
@@ -78,8 +79,8 @@ const prepend = (arch, urlPath, version) => {
     : `${endpoint}?platform=${platform}&arch=${arch}`
 }
 
-const getUrl = (arch, version) => {
-  if (is.url(version)) {
+const getUrl = (arch: string, version: string): string => {
+  if (is.webUrl(version)) {
```

Comment:
weirdly url is an alias of webUrl 🙄 so no change of behavior here but the type is right